kvirt/kvm/__init__.py

```diff
@@ -938,7 +938,10 @@ def console(self, name, tunnel=False):
                 url = "%s://%s:%s" % (protocol, host, localport)
                 if self.debug:
                     print(url)
-                consolecommand += "remote-viewer %s &" % url
+                if tunnel:
+                    consolecommand += "; remote-viewer %s &" % url
+                else:
+                    consolecommand += "remote-viewer %s &" % url
```

Comment:
you could have written it something like this btw:

consolecommand += ";remote-viewer %s &" % url if tunnel else "remote-viewer %s &"

Reply:
your solution is much better and smart as you are :)

Reply:
lol, dont think so, it s just way easier to solve things when the code is yours